src/app/body/filter-page/create-filter/create-filter.component.ts

```diff
@@ -133,24 +136,28 @@ export class CreateFilterComponent implements OnInit {
 
  submit(){
    this.enableLoader=true;
-   this.addFilter();
+   this.createFilter();
 }
 
-addFilter(){
+
+
+
+createFilter(){
   const orgDomain = this.backendService.getOrganizationDomain();
   const callable = this.functions.httpsCallable('filters/createFilter');
   callable({FilterName:this.filterName, Description:this.description, Difficulty: this.difficulty, Priority: this.priority, Status: this.status, SprintNumber: this.sprintNumber, OrgDomain: orgDomain, TeamName: this.teamName, Assignee: this.assigneeName.value}).subscribe({
     next:() => {
-      this.enableLoader = false;
+      this.enableLoader = true;
```

Comment:
loader is already enabled true in line 138, remove it from here and make it false in the createFilterDone function